ReactAndroid/src/main/java/com/facebook/react/animated/PropsAnimatedNode.java

```diff
@@ -89,7 +89,12 @@ public final void updateView() {
       } else if (node instanceof StyleAnimatedNode) {
         ((StyleAnimatedNode) node).collectViewUpdates(mPropMap);
       } else if (node instanceof ValueAnimatedNode) {
-        mPropMap.putDouble(entry.getKey(), ((ValueAnimatedNode) node).getValue());
+        String stringValue = ((ValueAnimatedNode) node).getStringValue();
+        if (stringValue != null) {
```

Comment:
It's not very generic. I'd rather allow value to be an `Object` and check whether it's an instance of `Double` or `String`.

Reply:
True, but that requires wider changes in react-native, as the rest of the code depends on getValue returning a double.

Reply:
@osdnk  You can try changing value to Object and see that quite a lot of the Animated logic would need to change to even make it compile. Or do you think we should keep value as a double, and add objValue of type Object, which can be something else?

Reply:
Renamed stringValue to animatedObject, and changed String to id in obj-c and Object in java https://github.com/facebook/react-native/pull/18187/commits/0521b3e085864d632cb257310c4b34195fab92f3